core/src/channel_group/mod.rs

```diff
@@ -132,6 +134,15 @@ impl ChannelGroup {
             }
         });
     }
+
+    pub fn voice_count(&self) -> u64 {
+        let mut vc = 0;
+        let channel_stats = self.channels.iter().map(|c| c.get_channel_stats()).to_vec();
```

Comment:
You don't need to use .to_vec() here. That iterates over the iterator and puts every item into a new Vec allocation.

Right below you're iterating again, so you could just remove to_vec and iterate over channel_stats directly as it's already an iterator.

Also, iterators have a `.sum()` so you could probably just use that instead too.

Reply:
Alright I'll remove the `to_vec` thing. Also I tried using `.sum()` but I got trait errors and I wasn't able to import what was needed for some reason.
I'll try one more time though.